docs/argus_status_page.md

```diff
@@ -0,0 +1,94 @@
+# Artgus Status Page documentation
+
+## Motivation
+In order to fast understand the Argus systems reachability all users need to implement a simple status page showing the status of Argus API, Web UI reachability, and the Cluster status. 
```

Comment:
`In order to quickly understand the reachability of the Argus system for all users we need to implement a simple <...>`